Prace_domowe/Praca_domowa3/adrianna_grudzień/pd3.ipynb

```diff
@@ -0,0 +1,664 @@
+{
```

Comment:
<ol><li>Po co zbiór testowy? Nawet nie jest nigdzie użyty</li><li>Zebranie wyników w jedno miejsce mile widziane</li><li>Kurde, no jakby dummy był najlepszy, to serio something is no yes</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>